tests/ethpm/test_package.py

```diff
@@ -62,6 +71,21 @@ def test_get_contract_factory_throws_if_name_isnt_present(safe_math_package, w3)
         assert safe_math_package.get_contract_factory("DoesNotExist", w3)
 
 
+def test_get_contract_instance(deployed_safe_math):
+    safe_math_package, address = deployed_safe_math
+    contract_instance = safe_math_package.get_contract_instance("SafeMathLib", address)
+    assert contract_instance.address == address
```

Comment:
Generally best to use `eth_utils.is_same_address` in cases like this since one of these addresses could end up being checksummed (automatically) at some point at which this would break.